### PR TITLE
[releng] Adds pom.xml files to build the flow sample

### PR DESCRIPTION
**Pull request by florianbarbin**
* For now, tycho is configured to just build flow, flow.edit and
flow.design plugins

Change-Id: I1da9e09173a1c66082795c0a040936f67519de22
Signed-off-by: Florian Barbin <florian.barbin@obeo.fr>

### CODE REVIEW

File: features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="fr.obeo.dsl.designer.sample.flow.design.feature"
+      label="Flow Sample Feature"
+      version="1.0.0.qualifier">
+
+   <description url="http://www.example.com/description">
```
Comment (mbats) [inline features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml]: Should be set

File: features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      [Enter Feature Description here.]
+   </description>
+
+   <copyright url="http://www.example.com/copyright">
```
Comment (mbats) [inline features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml]: Should be set

File: features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      [Enter Copyright Description here.]
+   </copyright>
+
+   <license url="http://www.example.com/license">
```
Comment (mbats) [inline features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml]: Should be set

File: features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml (hunk 1 above)
Comment (florianbarbin) [reply features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml]: Done ! (see second commit)

File: features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml (hunk 2 above)
Comment (florianbarbin) [reply features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml]: Done ! (see second commit)

File: features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml (hunk 3 above)
Comment (florianbarbin) [reply features/fr.obeo.dsl.designer.sample.flow.design.feature/feature.xml]: Done ! (see second commit)

**Pull request merged**